app/src/main/res/values/strings.xml

```diff
@@ -5,6 +5,11 @@
     <string name="activity_login_title">Log in to Last.fm</string>
     <string name="enable_notification_access">Enable Notification Access</string>
     <string name="enable_notification_access_summary">Required for scrobbling from Apple Music</string>
+    <string name="label_now_playing">Now Playing Track</string>
+    <string name="label_not_playing">Nothing!</string>
+    <string name="label_not_logged_in">Not logged in!</string>
+    <string name="label_not_playing_message">Let\'s play music on Apple Music App!</string>
+    <string name="label_message_to_log_in">Log in to Last.fm and play music on Apple Music App!</string>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mataku/SunsetScrob/blob/d98de3fb9ea0ace0e5c0241379c958a7804774f9/app/src/main/res/values/strings.xml#L12"></span>The resource <code>R.string.label_message_to_log_in</code> appears to be unused</td>
    </tr>
  </tbody>
</table>